### PR TITLE
Define different k8s source presets, add option to set event list globally

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add option to set events list globally
- Define different k8s source presets
  - `k8s-recommendation-events` - enables all possible k8s recommendations
  - `k8s-all-events` - is the name of the previous configuration
  - `k8s-err-events` - enables only `error` events for resources that were watched previously

## Related issue(s)

- #710

### CODE REVIEW

File: helm/botkube/values.yaml [hunk 1]
```diff
@@ -446,7 +447,7 @@ communications:
       bindings:
         # -- Notification sources configuration for the webhook.
         sources:
-          - k8s-events
+          - k8s-err-events
```
Comment (mszostok) [inline helm/botkube/values.yaml:450]:
```suggestion
          - k8s-err-events
          - k8s-recommendation-events
```

File: pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml [hunk 2]
```diff
@@ -8,17 +8,15 @@ sources:
                 pod:
                     noLatestImageTag: false
                     labelsSet: true
+            events:
```
Comment (pkosiec) [inline pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml:11]: If now we have global `namespace.include`, should it be also visible in the golden file? I can't see it 🤔

**Review by pkosiec — APPROVED** [review]
LGTM - please apply your suggestion and also see my question 🙂 

Disclaimer: Didn't test it, as tests cover that functionality. Will test the whole notification presets functionality E2E

File: pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml (hunk 2 above)
Comment (mszostok) [reply pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml:11]: There was a typo (missing `s`), should be `namespaces` instead. Fixed 👍

**Pull request merged**